packages/earth-map/src/components/MenuItemSkeleton/MenuItemSkeleton.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018-2020 National Geographic Society
+ *
+ * Use of this software does not constitute endorsement by National Geographic
+ * Society (NGS). The NGS name and NGS logo may not be used for any purpose without
+ * written permission from NGS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import MuiListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Skeleton from '@material-ui/lab/Skeleton';
+
+export function MenuItemSkeleton() {
+  const titleWidth = Math.random() * 40 + 30;
```

Comment:
Why we use random?

Reply:
This components renders a placeholder animation while the view is in a loading state.
We used random here to create different sizes for the placeholder in list elements.